test/Activity-test.js

```diff
@@ -22,76 +21,123 @@ describe('Activity', function() {
       ]
     });
     user2 = new User({
-      "id": 2,
-      "name": "Jarvis Considine",
-      "address": "30086 Kathryn Port, Ciceroland NE 07273",
-      "email": "Dimitri.Bechtelar11@gmail.com",
-      "strideLength": 4.5,
-      "dailyStepGoal": 2000,
-      "friends": [
+      'id': 2,
+      'name': 'Jarvis Considine',
+      'address': '30086 Kathryn Port, Ciceroland NE 07273',
+      'email': 'Dimitri.Bechtelar11@gmail.com',
+      'strideLength': 4.5,
+      'dailyStepGoal': 2000,
+      'friends': [
         9,
         18,
         24,
         19
       ]
     })
-    userRepository = new UserRepository();
-    userRepository.users.push(user1, user2);
-    activity1 = new Activity({
-      "userID": 1,
-      "date": "2019/06/15",
-      "numSteps": 3684,
-      "minutesActive": 140,
-      "flightsOfStairs": 16
-    }, userRepository);
-    activity2 = new Activity({
-      "userID": 2,
-      "date": "2019/06/20",
-      "numSteps": 2856,
-      "minutesActive": 280,
-      "flightsOfStairs": 22
-    }, userRepository);
+  
+    activity1 = new Activity(user1, date);
+    activity2 = new Activity(user2, date);
   });
   it('should be a function', function() {
     expect(Activity).to.be.a('function');
   });
   it('should be an instance of activity', function() {
     expect(activity1).to.be.an.instanceof(Activity);
   });
-  it('should hold a userId', function() {
-    expect(activity2.userId).to.equal(2);
-  });
-  it('should hold a date', function() {
-    expect(activity1.date).to.equal("2019/06/15");
-  });
-  it('should hold number of steps', function() {
-    expect(activity1.steps).to.equal(3684);
-  });
-  it('should hold minutes active', function() {
-    expect(activity2.minutesActive).to.equal(280);
-  });
-  it('should hold flights of stairs', function() {
-    expect(activity2.flightsOfStairs).to.equal(22);
-  });
-  it('should have a default value of 0 for miles walked', function() {
-    expect(activity2.milesWalked).to.equal(0);
+  it('should hold a user id, and date', function() {
+    expect(activity1.id).to.equal(1);
+    expect(activity1.date).to.equal('2019/03/05');
   });
-  it('should have a default value of null for reached step goal', function() {
-    expect(activity2.reachedStepGoal).to.equal(null);
+  it('should have a default value for reached step goal', function() {
+    expect(activity1.reachedStepGoal).to.equal(false);
   });
-  it('doActivity should add activities to user record', function() {
-    expect(user1.activityRecord.length).to.equal(1);
+  it('should have a default activity record', function() {
+    expect(activity1.activityRecord).to.deep.equal([]);
   });
-  it('should have a method that calculate miles walked', function() {
-    expect(activity1.calculateMiles(userRepository)).to.equal('3.0');
-  });
-  describe('compareStepGoal', function() {
+  describe('Activity Methods', function () {
+    it('should find a user\'s steps, minutes active, and flights climbed', function() {
```

Comment:
SAD path: What if no user id is found? What should be returned?

Reply:
Good idea, let's implement this on a branch for sad Path testing additions